Remove json_dispatcher folder

apuyou opened the pull request:
Les fichiers .inc, on peut les mettre dans le dossier inc/ :)

apuyou committed:
Remove json_dispatcher folder

Les fichiers .inc, on peut les mettre dans le dossier inc/ :)

trecouvr commented:
Le call_user_func_named il peut être utilisé par d'autres trucs, c'est vachement utile. Je l'aurais mis dans un truc genre utils.php ou dans une classe static Utils qu'on fou dans class/ et qui peut être autoload du coup.

On mettrait pas un sous dossier dans inc ? vu que pour le coup error_mapping et services je pense pas qu'il vont être inclus ailleurs.

apuyou commented:
Ok pour la classe `Utils`.

Pour le reste, est-ce que une classe `DispatcherJson` et une `DispatcherSoap` ça te va ?

apuyou committed:
PSR-0 style

J'ai créé le dossier src/ pour mettre les sources respectant le PSR-0, créé la classe Utils et mis les dispatchers (JSON et SOAP) dans des classes.

apuyou commented:
Je kiffe PSR-0 !

trecouvr commented on src/Payutc/Mapping/Errors.php:
```diff
@@ -0,0 +1,22 @@
+        	'ServiceMethodNotFound' => 404,
+        	'ServiceMethodForbidden' => 403,
+        	'ServiceMissingMethodArgument' => 400,
+        	'PossException' => function(PossException $e) {
```
Mettre \PossException pour sortir du namespace \Payutc\Mapping

apuyou committed:
PossException n'a pas de namespace
